torchvision/transforms/functional_pil.py

```diff
@@ -121,7 +126,13 @@ def adjust_gamma(img, gamma, gain=1):
 
 
 @torch.jit.unused
-def pad(img, padding, fill=0, padding_mode="constant"):
+def pad(
+    img: Image.Image,
+    padding: Union[int, List[int], Tuple[int, ...]],
+    fill: Optional[Union[numbers.Number, List[numbers.Number], Tuple[numbers.Number, ...]]],
```

Comment:
1. `mypy` treats `int` as virtual subtype of `float`. So if we substitute `numbers.Number` with `float`, we should be good even if we set the default argument to `0`.
2. Tests fail here, because we missed the ` = 0` default argument.

Reply:
Yeah I just spotted a minute ago too. :smile: (Coincidence)

I will cover these small fixes up and in a follow up PR ensure that the annotations between `functional_pil` and `functional_tesnor ` match properly. Not just on the pad case.